File: app/models/student.rb
class Student < ActiveRecord::Base
  self.table_name = "Students"
  self.primary_key = "StudentId"

  MAPPINGS = [
      [:FirstName, :first_name],
      [:LastName, :last_name],
      [:BirthDate, :birth_date],
      [:Address, :address],
      [:Suburb, :suburb],
      [:Postcode, :postcode],
      [:State, :state],
      [:Phone, :phone],
      [:Email, :email],
      [:UpdatedTS, :updated_at],
      [:CreatedTS, :created_at]
  ]

  MAPPINGS.each do |columns|
    alias_attribute columns[1], columns[0]
  end
end

File: frontend/app/models/student.js
import DS from 'ember-data';

export default DS.Model.extend({
  firstName: DS.attr('string'),
  lastName: DS.attr('string'),
  birthDate: DS.attr(),
  address: DS.attr('string'),
  suburb: DS.attr('string'),
  state: DS.attr('string'),
  postcode: DS.attr('string'),
  phone: DS.attr('string'),
  email: DS.attr('string'),
  updatedAt: DS.attr(),
  createdAt: DS.attr()
});
